source/Star.js

```diff
@@ -0,0 +1,23 @@
+var Pixi = require("pixi.js")
+
+import Reference from "./Reference.js"
+
+export default class Star extends Pixi.Sprite {
+    constructor(parallax) {
+        super(PIXI.loader.resources.star.texture)
+        this.position.x = Math.random() * Reference.GAME_WIDTH
+        this.position.y = Math.random() * Reference.GAME_HEIGHT
+        this.tint = Reference.STAR_COLORS[Math.floor(Math.random() * Reference.STAR_COLORS.length)]
+        this.alpha = parallax * .1
+        this.speed = parallax * .4
+    }
+
+    update(delta) {
+        this.position.x -= this.speed
+        if (this.position.x < -this.width) {
+            this.position.x *= -1
+            this.position.x += Reference.GAME_WIDTH
```

Comment:
This is an interesting way to cycle the stars back to the right side of the screen. What's the *= -1 for? Looks like it moves the x position to 1, then adds the game width to it. Why not just set the position to the game width?